PyDSS/reports/voltage_metrics.py

```diff
@@ -102,6 +104,8 @@ def generate(self, output_dir):
             f_out.write("\n")
 
         logger.info("Generated %s", filename)
+        for filename in self._files_to_delete:
+            os.remove(filename)
```

Comment:
check `os.path.exists(filename)` before remove?

Reply:
That should be an impossible condition. (Unless you see something I don't)